website/src/controllers/groups.js

```diff
@@ -24,7 +24,7 @@ var api = module.exports;
 var partyFields = api.partyFields = 'profile preferences stats achievements party backer contributor auth.timestamps items';
 var nameFields = 'profile.name';
 var challengeFields = '_id name';
-var guildPopulate = {path: 'members', select: nameFields, options: {limit: 15} };
+var guildPopulate = {path: 'members', select: nameFields, options: {limit: 2} };
```

Comment:
Restricting this to two members means that only the guild leader and the current user show up.

![screenshot 2015-04-02 06 07 17](https://cloud.githubusercontent.com/assets/10201/6962056/f645aee8-d8fe-11e4-969c-d952eb80ae4f.png)

Looks like the current user still shows up with this capped at 15 users. Is restricting the displayed group members desired behavior?